.env.defaults

```diff
@@ -18,6 +18,7 @@ SOCKETY_SERVER_API=wss://sockety.odysee.com/ws
 RECSYS_ENDPOINT=https://recsys.odysee.com/log/video/view
 THUMBNAIL_CDN_URL=https://thumbnails.odycdn.com/optimize/
 THUMBNAIL_CARDS_CDN_URL=https://cards.odycdn.com/
+LOCALE_API=https://api.odysee.com/locale/get
```

Comment:
I selected one item but the modal still displayed 2 options.
Not a big deal, although seems to defeat the purpose of the drop-down.

<img width="500" alt="image" src="https://user-images.githubusercontent.com/64950861/156005795-369183bf-fe02-4a20-9ec6-05ebeebc1c8c.png">

Reply:
There's also only a single chinese homepage, and I believe we load it for both zh-hans and hant. So there shouldn't be two options.